### DISCUSSION TITLE
Multiple startup/restart configurations

### DISCUSSION THREAD

**Discussion by 18923** [blakeblackshear/frigate#4678] [Ideas]
Start Frigate with selected frigate.yaml file, for example when I am home I only want to keep recorded files for 1 or two days when away for a few days keep files for several days. I have a Hickvison NVR for long term storage, I'm using Frigate to facilitate searching for events. I can do this now by stopping Frigate rename the yaml file and restart. 

For example in the "Restart Frigate" drop down there could be a selection of configurations to start from, FrigateHome. yaml, FrigateAway.yaml, FrigateOutsideCams.yaml, etc . . .

**Comment by tbrausch** [comment]
See [(https://github.com/blakeblackshear/frigate/issues/1911)]
